### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by simplerick** [Einsturing/StemGNN-pytorch#1]
Hi, thank you for your work! 
Does your code reproduce the results of the official implementation https://github.com/microsoft/StemGNN? 
Also am I right that your version supports channel dimension, i.e input can have 4 dims instead of 3 as in the official version?

**Comment by Einsturing** [comment]
Hello, thanks for your issue.
This version was implemented based on the official branch 'juaduan/dev'. I change the TensorFlow implementation to Pytorch.
I did not use my code to reproduce the official results because they do not support my research, but in my research's datasets, it can perform well, allowing me to compare it with my own methods. However, I can't reproduce its own results using official code, whether in the paper or GitHub, in fact, it shows no signs of convergence at all, which is also the reason why I have to reimplement it. 
And yes, input can have 4 dims, but it's not my contribution, the official 'juaduan/dev' branch can support 4 dims.
I hope this can help you!
